src/utils.jl

```diff
@@ -0,0 +1,24 @@
+sprintcompact(x) = sprint(show, x; context=:compact => true)
```

Comment:
Was this inadvertently revived by a rebase ? see https://github.com/JuliaWeb/HTTP.jl/commit/0e548d1a6ff1ebbb20ba89be179105f57ec74330

Reply:
Hmmm, must have. Fixed now.